raiden/utils/__init__.py

```diff
@@ -136,14 +135,15 @@ def host_port_to_endpoint(host: str, port: int) -> str:
     return '{}:{}'.format(host, port)
 
 
-def split_endpoint(endpoint: str) -> Tuple[str, Union[str, int]]:
+def split_endpoint(endpoint: str) -> Tuple[str, Union[str, Optional[int]]]:
```

Comment:
Please, please, simplify the type here. E.g. `Tuple[Host, Optional[Port]]`

Reply:
Sure I can. The purpose of the PRs is to satisfy mypy so some times types can get complicated. Let me see if I can improve this particular case.